src/anemoi/training/diagnostics/mlflow/logger.py

```diff
@@ -309,6 +309,7 @@ def __init__(
 
         self._fork_run_server2server = None
         self._parent_run_server2server = None
+        tags = None
```

Comment:
Why does tags need to be initialised to None?

Reply:
Since now the `_get_mlflow_run_params` is a method o `AnemoiMLflowLogger`. Tags are not passed as an input the `AnemoiMLflowLogger` but rather created with this function in the __init__ method. `_get_mlflow_run_params` is just called in rank0 since it's that rank the one that handles of the logging and writing of artifacts, plots and checkpoints. So for all the other ranks we need a default value for the tags to avoid the code failing. (note `run_id` and `run_name` don't need that cause we pass that from the config defaults when defining the `AnemoiMLflowLogger`